### DISCUSSION TITLE
switchesResult Hook

### DISCUSSION THREAD

**Discussion by Yahav** [PaperStrike/Pjax#248] [Q&A] [answered]
Is the switchesResult hook executed after the new javascripts were executed?

**Comment by PaperStrike** [comment, marked as answer]
No, the `switchesResult` hook is executed before the new scripts.

**Reply by Yahav** [reply]
Is there a hook for after the new scripts executed?

**Reply by PaperStrike** [reply]
There's no hook for that. Hooks are to modify something, while nothing will be produced by a normal script execution nor can be observed by Pjax. Will events like `pjax:complete`/`pjax:success` suit your need?
